src/coreclr/interpreter/compiler.cpp

```diff
@@ -1366,15 +1558,23 @@ bool InterpCompiler::CreateBasicBlocks(CORINFO_METHOD_INFO* methodInfo)
             target = insOffset + 2 + (int8_t)ip [1];
             if (target >= codeSize)
                 return false;
-            GetBB(target);
+            pLeaveTargetBB = GetBB(target);
+            if (opcode == CEE_LEAVE_S || opcode == CEE_LEAVE)
```

Comment:
If it's a short target it's always LEAVE_S right?

Reply:
You are right, checking just the _S one here and the non _S one below is sufficient.